src/pages/Apply.tsx

```diff
@@ -0,0 +1,67 @@
+import { Fragment } from 'react/jsx-runtime';
+
+import BlockButton from '@/components/common/button/BlockButton';
+import Icon from '@/components/common/icon/Icon';
+import ProgressItem from '@/components/common/progress/ProgressItem';
+import ProgressVerticalBridge from '@/components/common/progress/ProgressVerticalBridge';
+import Title from '@/components/common/Title';
+import { applyInfoList, applyProcedureList } from '@/constants/applyData';
+
+function Apply() {
+  // TODO: 개인정보 맟 이용 동의서, 회비 링크 걸기
+  const now = new Date();
```

Comment:
사소한 부분일 수 있는데, 해당 함수가 렌더링 마다 다시 실행되는 경우도 있을 거 같아요. 이 부분도 고려하면 좋을 거 같습니다!

```suggestion
  const currentDate = new Date();
```
또한 이런 네이밍도 좋아보이네요..!

Reply:
원준님 말씀 듣고 new Date를 한번만 호출할 수는 없는지 고민을하다가 
생각해보니 만약에 1초 후 날짜가 바뀌면 바뀐 날짜를 반영해야하니까 new Date는 실행되도록 두어야될 거 같단..생각입니다! 

네이밍은 currentDate가 더 적절해보여서 수정해뒀습니다 ㅎㅎ